pkg/apis/scaler/v1alpha2/crd_demand.go

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) 2019 Palantir Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha2
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// DemandPhaseEmpty is the state of a demand object when it is first created
+	DemandPhaseEmpty string = ""
```

Comment:
Can you make the `v1alpha1` consts point to these?

Reply:
I can make the import v1alpha2 inside v1alpaha1 not the other way round. Will do it

Reply:
I meant make v1alpha1 import v1alpha2.